geti_sdk/annotation_readers/geti_annotation_reader.py

```diff
@@ -160,6 +161,18 @@ def get_data(
                             label_name=label_dict["name"]
                         )
             new_annotations.append(annotation_object)
+            if "anomal" in annotation_object.labels[0].name.lower():
+                # Anomaly reduction: Convert anomalous annotations to full rectangles
```

Comment:
I'd add a bit more context here, to remind our future selves what the purpose of this block is. Something like:
 
`# Part of anomaly task reduction in Geti 2.5 -> all anomaly tasks combined into one. Geti now only accepts full rectangles for anomaly tasks`

Reply:
added the comment